app/controllers/nonsipp/memberdetails/upload/FileUploadSuccessController.scala

```diff
@@ -88,7 +99,21 @@ class FileUploadSuccessController @Inject()(
       UserAnswers.remove(DoesMemberHaveNinoPages(srn)),
       UserAnswers.remove(MemberDetailsNinoPages(srn)),
       UserAnswers.remove(NoNinoPages(srn)),
-      UserAnswers.remove(MemberStatuses(srn))
+      UserAnswers.remove(MemberStatuses(srn)),
```

Comment:
FYI @westwater this PR may gets merged before the soft delete changes. Can you see anything that wouldn't fit with your approach of soft delete?

Reply:
Yeah It looks like this will conflict with soft delete - because of how the soft delete is working, I need to hard delete all member journeys when a member gets deleted which look like part of what this PR is trying to do

Reply:
Thanks @westwater . Do you want @B-Savage to go-ahead with these changes as they are and you're going to adjust them to the soft delete, or do you want @B-Savage to change the approach?
AFAIK hard delete of members is still valid in some scenarios (e.g. compiled status, version1)

Reply:
Yeah lets get this in and i'll deal with it on my PR